services/index/internal/datasources/elasticsearch/index.go

```diff
@@ -21,23 +21,16 @@ var indices = []index{
 			   "_source":{
 				  "includes":[
 					 "geolocation",
-					 "keywords",
 					 "lastChecked",
 					 "linkedSchemas",
 					 "maplocation",
-					 "mission",
-					 "name",
-					 "profileUrl",
-					 "url"
+					 "profileUrl"
```

Comment:
Since the output contains `porifleURl` it's better just storing it. 

Otherwise, we have to query from MongoDB and it makes the logic complex.

![image](https://user-images.githubusercontent.com/11765228/98504465-0044a380-2292-11eb-8b83-9a6b3d08cad6.png)